controllers/provider-aws/pkg/controlplane/actuator.go

```diff
@@ -0,0 +1,359 @@
+	Objects: []*chart.Object{
+		{
+			Type: &corev1.ConfigMap{},
+			Name: common.CloudProviderConfigName,
+		},
+	},
+}
+
+const (
+	// AccessKeyID is a constant for the key in a cloud provider secret and backup secret that holds the AWS access key id.
+	AccessKeyID = "accessKeyID"
+	// SecretAccessKey is a constant for the key in a cloud provider secret and backup secret that holds the AWS secret access key.
+	SecretAccessKey = "secretAccessKey"
+)
+
+var ccmChart = &chart.Chart{
+	Name:   "cloud-controller-manager",
+	Path:   filepath.Join(aws.ChartsPath, "cloud-controller-manager"),
+	Images: []string{common.HyperkubeImageName},
+	ValuesFunc: func(clusterName string, shoot *gardenv1beta1.Shoot, checksums map[string]string) (map[string]interface{}, error) {
+		s := &extensionscontroller.Shoot{Shoot: shoot}
+		return map[string]interface{}{
+			"cloudProvider":     "aws",
+			"clusterName":       clusterName,
+			"kubernetesVersion": s.GetKubernetesVersion(),
+			"podNetwork":        s.GetPodNetwork(),
+			"replicas":          s.GetReplicas(1),
+			"podAnnotations": map[string]interface{}{
+				"checksum/secret-cloud-controller-manager":        checksums[common.CloudControllerManagerDeploymentName],
+				"checksum/secret-cloud-controller-manager-server": checksums[common.CloudControllerManagerServerName],
+				"checksum/secret-cloudprovider":                   checksums[common.CloudProviderSecretName],
+				"checksum/configmap-cloud-provider-config":        checksums[common.CloudProviderConfigName],
+			},
+			"configureRoutes": false,
+			"environment": []map[string]interface{}{
+				{
+					"name": "AWS_ACCESS_KEY_ID",
+					"valueFrom": map[string]interface{}{
+						"secretKeyRef": map[string]interface{}{
+							"key":  AccessKeyID,
+							"name": common.CloudProviderSecretName,
+						},
+					},
+				},
+				{
+					"name": "AWS_SECRET_ACCESS_KEY",
+					"valueFrom": map[string]interface{}{
+						"secretKeyRef": map[string]interface{}{
+							"key":  SecretAccessKey,
+							"name": common.CloudProviderSecretName,
+						},
+					},
+				},
+			},
+			// TODO Use the higher resource limits below only for the "shooted seed" case
+			"resources": map[string]interface{}{
+				"limits": map[string]interface{}{
+					"cpu":    "500m",
+					"memory": "512Mi",
+				},
+			},
+		}, nil
+	},
+	Objects: []*chart.Object{
+		{Type: &corev1.Service{}, Name: "cloud-controller-manager"},
+		{Type: &appsv1.Deployment{}, Name: "cloud-controller-manager"},
+	},
+}
+
+// NewActuator creates a new Actuator that acts upon and updates the status of ControlPlane resources.
+func NewActuator() controlplane.Actuator {
+	return &actuator{
+		logger: log.Log.WithName("controlplane-controller"),
+	}
+}
+
+// actuator is an Actuator that acts upon and updates the status of ControlPlane resources.
+type actuator struct {
+	scheme       *runtime.Scheme
+	decoder      runtime.Decoder
+	config       *rest.Config
+	cs           kubernetes.Interface
```

Comment:
```suggestion
	clientset    kubernetes.Interface
```

Reply:
Done, `gcs` renamed to `gardenerClientset` as well.